base-build/Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM alpine:3.16.0
 
-USER root
-
-RUN apk upgrade --no-cache && apk add --no-cache openjdk17 curl diffutils
```

Comment:
Do we not need diffutils anymore? Thought workflow still required it for its tests?

Reply:
We removed the tests completely https://github.com/UKHomeOffice/hocs-workflow/pull/699